netlink-request/Cargo.toml

```diff
@@ -4,9 +4,10 @@ version = "1.5.5"
 edition = "2021"
 
 [target.'cfg(target_os = "linux")'.dependencies]
-netlink-sys = "0.8"
-netlink-packet-core = "0.4"
-netlink-packet-generic = "0.3"
-netlink-packet-route = "0.13"
+netlink-sys = "0.8.5"
+netlink-packet-core = "0.5"
+netlink-packet-generic = "0.3.2"
+netlink-packet-route = "0.15.0"
+netlink-packet-utils = "0.5.2"
```

Comment:
Was there something released which requires us to specify the patch versions?

Reply:
Yeah this reminds me a discussion with @goodhoko in https://github.com/tonarino/acoustic_profiler/pull/12#discussion_r1172379880.

As innernet is shipped as a binary and not a library, I don't think we need to aim for maximum possible version range for dependencies (which is tedious if done correctly, per reasoning in the linked thread). So I'm cool with specifying also the patch versions. It reduces the combinatorial space of possible build configurations.

Only thing that tickles my OCD is saying `0.15.0` instead of `0.15` (these 2 are strictly equivalent), but OTOH that is consistent with other lines..

Reply:
I've gotten into the habit after discussions at work about this - specifying a higher and more specific semver prevents any bugs in previous versions from showing up when cargo decides for whatever reason to use it instead. For example, I actually had a bug with an earlier patch release in the netlink crate family not compiling.

Happy to get rid of that extra '0' for you @strohel :).